src/actions/reflexer/ReflexerNativeUniV2SaviourDepositAction.js

```diff
@@ -0,0 +1,29 @@
+const Action = require("../../Action");
+
+const { getAddr } = require('../../addresses.js');
+
+/**
+ * ReflexerNativeUniV2SaviourDepositAction - 
+ */
+class ReflexerNativeUniV2SaviourDepositAction extends Action {
+  /**
+   * @param from {EthAddress}
+   * @param safeId {SafeId}
+   * @param lpTokenAmount {string}
+   */
+  constructor(from, safeId, lpTokenAmount, lpTokenAddress) {
+    super('ReflexerNativeUniV2SaviourDeposit', getAddr('ReflexerNativeUniV2SaviourDeposit'), [['address','uint256','uint256']], [[from, safeId, lpTokenAmount]]);
+    this.mappableArgs = [
+        this.args[0][0],
+        this.args[0][2],
```

Comment:
If someone created in the same recipe a new reflexer safe, there will be no way to pipe that id?